### PR TITLE
Fix anime split show layout

### PR DESCRIPTION
**Pull request by h3llrais3r**
Fixes https://github.com/pymedusa/Medusa/issues/3532

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fernandog** [comment]
@h3llrais3r can you send screenshots?

File: views/partials/home/banner.mako [hunk 1]
```diff
@@ -13,8 +13,10 @@
 % for cur_show_list in show_lists:
     <% cur_list_type = cur_show_list[0] %>
     <% my_show_list = list(cur_show_list[1]) %>
-    % if cur_list_type == "Anime":
-        <h1 class="header">Anime List</h1>
+    % if len(show_lists) > 1:
+        % if len(show_lists) > 1:
```
Comment (medariox) [inline views/partials/home/banner.mako]: Why the double check here? 😕

File: views/partials/home/simple.mako [hunk 2]
```diff
@@ -15,8 +15,10 @@
 % for cur_show_list in show_lists:
     <% cur_list_type = cur_show_list[0] %>
     <% my_show_list = list(cur_show_list[1]) %>
-    % if cur_list_type == "Anime":
-        <h1 class="header">Anime List</h1>
+    % if len(show_lists) > 1:
+        % if len(show_lists) > 1:
```
Comment (medariox) [inline views/partials/home/simple.mako]: And here

File: views/partials/home/banner.mako (hunk 1 above)
Comment (p0psicles) [reply views/partials/home/banner.mako]: I'll fix that. Sorry. Didn't notice. Still have it checked out anyway.

Comment (medariox) [reply views/partials/home/banner.mako]: Np, thanks!

File: views/partials/home/simple.mako (hunk 2 above)
Comment (h3llrais3r) [reply views/partials/home/simple.mako]: Sorry for that, copy paste error... :roll_eyes:

**Pull request merged**